src/main/resources/site/parts/omStatistikken/omStatistikken.ts

```diff
@@ -0,0 +1,239 @@
+import { formatDate } from '../../../lib/ssb/utils/dateUtils'
+import { Content } from 'enonic-types/content'
+import { Request, Response } from 'enonic-types/controller'
+import { React4xp, React4xpResponse } from '../../../lib/types/react4xp'
+import { Phrases } from '../../../lib/types/language'
+import { Statistics } from '../../content-types/statistics/statistics'
+import { OmStatistikken } from '../../content-types/omStatistikken/omStatistikken'
+import { ReleaseDatesVariant, StatisticInListing, VariantInListing } from '../../../lib/ssb/dashboard/statreg/types'
+
+const {
+  get
+} = __non_webpack_require__('/lib/xp/content')
+const {
+  getContent, processHtml
+} = __non_webpack_require__('/lib/xp/portal')
+const {
+  renderError
+} = __non_webpack_require__('/lib/ssb/error/error')
+const {
+  getStatisticByIdFromRepo
+} = __non_webpack_require__('/lib/ssb/statreg/statistics')
+const {
+  getPhrases
+} = __non_webpack_require__('/lib/ssb/utils/language')
+const {
+  getReleaseDatesByVariants
+} = __non_webpack_require__('/lib/ssb/statreg/statistics')
+const {
+  fromPartCache
+} = __non_webpack_require__('/lib/ssb/cache/partCache')
+const {
+  data: {
+    forceArray
+  }
+} = __non_webpack_require__('/lib/util')
+const React4xp: React4xp = __non_webpack_require__('/lib/enonic/react4xp') as React4xp
+
+
+exports.get = function(req:Request):Response | React4xpResponse {
+  try {
+    const statisticPage: Content<Statistics> = getContent()
+    return renderPart(req, statisticPage.data.aboutTheStatistics)
+  } catch (e) {
+    return renderError(req, 'Error in part', e)
+  }
+}
+
+exports.preview = (req: Request, id: string | undefined):Response | React4xpResponse => renderPart(req, id)
+
+function renderPart(req:Request, aboutTheStatisticsId: string | undefined):Response | React4xpResponse {
+  const page: Content<any> = getContent()
```

Comment:
Is <any> needed? Might work with just Content (applies to getOmStatistikken func params as well)